### PR TITLE
Rename seedu.address

### LINKED ISSUE #40: Rename package seedu.address
From https://github.com/AY2021S1-CS2103T-W16-3/tp/pull/39#issuecomment-699654359

Refactoring the package will generate a large diff, making it difficult to review any other changes. It should be fixed in a PR of its own.

### PR DESCRIPTION
**Pull request by zhaojj2209**
Resolves #40.

Refactors the `seedu.address` package to `ay2021s1_cs2103_w16_3.finesse`.

### CODE REVIEW

File: src/test/resources/view/UiPartTest/validFileWithFxRoot.fxml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<fx:root type="seedu.address.ui.TestFxmlObject" xmlns="http://javafx.com/javafx"
-            xmlns:fx="http://javafx.com/fxml">
+<fx:root type="ay2021s1_cs2103_w16_3.finesse.ui.TestFxmlObject" xmlns="http://javafx.com/javafx"
+           xmlns:fx="http://javafx.com/fxml">
```
Comment (wltan) [inline src/test/resources/view/UiPartTest/validFileWithFxRoot.fxml:4]: Indentation still appears to be slightly off here compared to the previous state
```suggestion
            xmlns:fx="http://javafx.com/fxml">
```

**Review by wltan — CHANGES_REQUESTED** [review]

**Comment by wltan** [comment]
Notes:
- Checkstyle has been modified to accept the new package name.
- Packages inside the `docs` folder have not been changed, as documentation will be revised towards the end of v1.2.

**Review by wltan — APPROVED** [review]
LGTM! Notes:
- Checkstyle has been modified to accept the new package name.
- Packages inside the `docs` folder have not been changed, as documentation will be revised towards the end of v1.2.

**Review by yongping827 — APPROVED** [review]
LGTM

**Pull request merged**